src/i18n/index.js

```diff
@@ -4,15 +4,7 @@ import lngDetector from 'i18next-browser-languagedetector'
 import VueI18Next from '@panter/vue-i18next'
 import moment from 'moment'
 
-// All Calendar locales we need
-// see https://github.com/fullcalendar/fullcalendar/tree/master/locales for full list
-// @todo test if and how this even works!?
-import 'fullcalendar/dist/locale/de.js'
-import 'fullcalendar/dist/locale/es.js'
-import 'fullcalendar/dist/locale/fr.js'
-import 'fullcalendar/dist/locale/it.js'
-import 'fullcalendar/dist/locale/es-us.js'
-import 'fullcalendar/dist/locale/en-gb.js'
+import '@fullcalendar/core/locales-all'
```

Comment:
what is under all? :)

Reply:
All of them (https://github.com/fullcalendar/fullcalendar/tree/master/packages/core/src/locales)

I did another look in the docs, and found that they now provide a nicer way; it will require a re-render, but better than this.

Reply:
@darh https://github.com/cortezaproject/corteza-webapp-compose/pull/137/commits/da35c1df755f5f98d4af5002e68bf88a0de9b8ee